lib/block-patterns.php

```diff
@@ -5,15 +5,18 @@
  * @package gutenberg
  */
 
-register_block_pattern_category( 'Query', array( 'label' => __( 'Query', 'gutenberg' ) ) );
+// Register categories used for block patterns.
+register_block_pattern_category( 'query', array( 'label' => __( 'Query', 'gutenberg' ) ) );
+register_block_pattern_category( 'text', array( 'label' => __( 'Text', 'gutenberg' ) ) );
+register_block_pattern_category( 'buttons', array( 'label' => __( 'Buttons', 'gutenberg' ) ) );
```

Comment:
I was registering these inside the init filter because if I didn't I was getting an error while the admin was loading, but I don't see that any more

Reply:
I'm not sure registering these categories is needed, except for `query` that doesn't exists in `core`. Also not introduced in this PR, but the registration of the other categories might not be needed.. Will look now..